### ISSUE TITLE
Caps lock LED not working

### ISSUE THREAD

**Issue by Patsonical** [rvaiya/keyd#305]
When `layer_indicator` is set to 0, the caps lock led is always off. Is there a way to restore the normal indicator behaviour, where the led is lit when caps lock is on?

**Comment by rvaiya** [comment]
Do you mean when `layer_indicator` is set to 1? Leaving it unset should have no effect on the caps lock indicator (unless by 'capslock' you actually mean `toggling the shift layer`)). Can you post the output of `sudo keyd -m` when you are activating what you believe to be capslock?

**Comment by Patsonical** [comment]
No, when it's set to 1 then it works as it's meant to in that mode, but when set to 0 it doesn't work at all. Here is the output of the monitor command as I press capslock (twice):
```
patsonical@pats-nixos ~ [2]> sudo keyd -m
[sudo] password for patsonical:
device added: 0fac:1ade (keyd virtual pointer)
device added: 0fac:0ade (keyd virtual device)
device added: 06cb:cd8c (SYNA8005:00 06CB:CD8C Mouse)
device added: 0002:000a (TPPS/2 Elan TrackPoint)
device added: 0001:0001 (AT Translated Set 2 keyboard)
keyd virtual device	0fac:0ade	enter up
keyd virtual device	0fac:0ade	capslock down
keyd virtual device	0fac:0ade	capslock up
keyd virtual device	0fac:0ade	capslock down
keyd virtual device	0fac:0ade	capslock up
```
This is with a config that doesn't modify capslock's behaviour (just as a test), but I get the same result if I rebind (e.g.) shift+capslock to capslock:
```
[ids]
*

[global]
layer_indicator = 0

[main]
9 = 0
```

**Comment by rvaiya** [comment]
Can you try the latest commit?

**Comment by rvaiya** [comment]
To clarify, are you suggesting that it works if you completely remove `layout_indicator` from your config? Or simply that it doesn't work at all unless you set it to 1?

**Comment by Patsonical** [comment]
It doesn't work at all when keyd is running, unless `layout_indicator` is set to `1` (in which case it uses the keyd layer behaviour). Otherwise it doesn't work as a regular capslock indicator unless I kill keyd entirely. Also, I just tried the indicator in Waybar, and that doesn't show capslock as locked either when keyd is running (it works normally without keyd). It's not "game-breaking", but it's an odd bug.

**Comment by Kuchteq** [comment]
For me it doesn't work only on window managers (dwm and dwl) and ttys, kde and gnome show the light accordingly, tested on two laptops with a built in keyboard.

**Comment by kvaster** [comment]
I have the same problem. I'm using kde plasma. Also I'm using caps lock to switch languages and caps lock led as an language indicator. Without keyd running it's all OK. With keyd:
* if I press caps lock - language changes, but caps lock is not triggered.
* if I click on language indicator in the bar - language changes and caps lock led is also activated/deactivated according to language
* if caps lock led is activated and I press caps lock - language will change, but caps lock led will be activated all the time.

My config:
```
[ids]
*

[main]
sysrq = insert
```
